### DISCUSSION TITLE
"Edit" links fail to work on the "Claim types configuration" page

### DISCUSSION THREAD

**Discussion by HermanGyorgy** [Yvand/EntraCP#218] [General]
Installed EntraCP 23.0.20231121 for testing purposes.  
In my test environment experience the below issue:  
 
"Edit" links fail to work on the "Claim types configuration" (/_admin/EntraCP/ClaimTypesConfig.aspx) page:
("Delete" links on the same page works OK.)

Edit link associated with below script:
javascript:Azurecp.ClaimsTablePage.EditItem('<n>')
<n>: is some index number

Error displayed on (F12) Developer Tools console tab:
SCRIPT5009: 'Azurecp' is undefined

**Comment by Yvand** [comment]
Thank you for reporting this, I will take a look and report here

**Reply by Yvand** [reply]
It is fixed with https://github.com/Yvand/EntraCP/pull/220
Thanks again
